### ISSUE TITLE
RL performs poorly than SFT in the geometry

### ISSUE THREAD

**Issue by mingliangzhang2018** [Deep-Agent/R1-V#154]
Dear authors,

Thank you for opening project source. It is extremely meaningful.

You indicate that RL performs poorly than SFT in the geometry. This phenomenon may attribute to data duplication in GeoQA dataset. In the training set and  test set, many problems have the same textual representation but only different numerical values. Also, there are a lot of duplicate diagrams that are not exactly the same numerically. Therefore, this dataset is easy to fit through the SFT. 

I recommend using the PGPS9K dataset to validate whether RL promotes geometric visual perception and geometric reasoning. The PGPS9K (https://github.com/mingliangzhang2018/PGPS) is a visual-domain geometry dataset with less textual bias.

Sincerely.
